multigeiger/switches.h

```diff
@@ -4,9 +4,14 @@
 #define _SWITCHES_H_
 
 // What are the switches good for?
-enum {SPEAKER_ON, DISPLAY_ON, LED_ON, UNUSED};
+typedef struct switches {
+  unsigned int speaker_on: 1;  // SW0
+  unsigned int display_on: 1;  // SW1
+  unsigned int led_on: 1;  // SW2
+  unsigned int unused: 1;  // SW3
+} Switches;
```

Comment:
sometimes even C has nice things.

Reply:
Yea, look good.
But is it better ini terms of code size?

Reply:
I didn't check, but I would not expect more than a few bytes difference, if any.

Reply:
master:

856326 bytes (25%) of program storage.
Global variables use 44912 bytes (13%) of dynamic memory.

this PR:

856338 bytes (25%) of program storage space.
Global variables use 44912 bytes (13%) of dynamic memory.

So, 12 bytes more program code.

Reply:
OK, that's negligible.